Tools/build/deepfreeze.py

```diff
@@ -441,12 +454,10 @@ def is_frozen_header(source: str) -> bool:
 
 
 def decode_frozen_data(source: str) -> types.CodeType:
-    lines = source.splitlines()
-    while lines and re.match(FROZEN_DATA_LINE, lines[0]) is None:
-        del lines[0]
-    while lines and re.match(FROZEN_DATA_LINE, lines[-1]) is None:
-        del lines[-1]
-    values: Tuple[int, ...] = ast.literal_eval("".join(lines).strip())
+    values: list[int] = []
+    for line in source.splitlines():
+        if re.match(FROZEN_DATA_LINE, line):
+            values.extend([int(x) for x in line.split(",") if x.strip()])
```

Comment:
```suggestion
            values.extend(int(x) for x in line.split(",") if x.strip())
```

Reply:
If you look through my commits, that's what I actually tried first, but the listcomp is slightly faster. Probably because of Carl's optimizations for PEP 709.

Reply:
My mistake, sorry -- shows the benefit of 709 though!